src/PhpSession.php

```diff
@@ -103,7 +103,7 @@ public function unauthorizedResponse(ServerRequestInterface $request) : Response
                 'Location',
                 $this->config['redirect']
             )
-            ->withStatus(301);
+            ->withStatus(302);
```

Comment:
Hm... or maybe it should be configurable? I can see we have `$this->config` here so users can chose what they want to chose.

Reply:
@wshafer is correct. As the 301 code is a permanent redirect, browser may (and will) cache it. I've experienced this as well.

Reply:
@webimpress Definitely needs to be a 302; it's **not** permanent, as after authentication, they should be able to proceed to the originally requested resource.